### ISSUE TITLE
[icu:x64-windows] build failure

### ISSUE THREAD

**Issue by Eschryn** [microsoft/vcpkg#11334]
**Host Environment**
- OS: Windows 10
- Compiler: Visual Studio Build Tools 2019 16.5.5

**To Reproduce**
`./vcpkg install libmysql:x64-windows`

**Failure logs** 
-err.log:
```
FEHLER: Auf dem Remotesystem muss Windows 2000 oder höher ausgeführt werden.
```

**Additional context**
I tried updating vcpkg

**Comment by cenit** [comment]
Is there any other message or error log available? Is it related to #11335 ?

**Comment by PhoebeHui** [comment]
Duplicated to https://github.com/microsoft/vcpkg/issues/11335.
